spring-cloud-huawei-router-client/src/main/java/com/huaweicloud/router/client/RouterClientAutoConfiguration.java

```diff
@@ -28,8 +28,8 @@
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
-import com.huaweicloud.router.client.loabalancer.CanaryServiceInstanceFilter;
-import com.huaweicloud.router.client.loabalancer.ZoneAwareServiceInstanceFilter;
+import com.huaweicloud.router.client.loadalancer.CanaryServiceInstanceFilter;
+import com.huaweicloud.router.client.loadalancer.ZoneAwareServiceInstanceFilter;
```

Comment:
loadbalancer not loadalancer,  missing the letter b

Reply:
nice catch! fixed.